src/testPane.ts

```diff
@@ -719,11 +719,16 @@ async function debugNode(
 
         createEmptyLaunchConfigFile(ourWorkspace, launchJsonPath);
         addLaunchConfiguration(launchJsonUri);
-      } else {
+      } 
+      else {
```

Comment:
Are we formatting this file at all?

Reply:
Have not, happy to run black on it as a separate branch

Reply:
Okay, cool.